ext/cfx-ui/src/assets/languages/locale-fr.json

```diff
@@ -1,113 +1,303 @@
+  "#Global_OpenLinkInBrowser": "Ouvrir le lien dans le navigateur",
+  "#AuthForm_Success_Welcome": "Bienvenue,",
+  "#AuthForm_Success_Continue": "Continuer",
+  "#AuthForm_External_Initialize": "Connexion via le navigateur",
+  "#AuthForm_External_Instruction": "Cliquez sur Autoriser dans le navigateur",
+  "#AuthForm_External_Or_Internal": "ou saisissez les identifiants",
+  "#AuthForm_External_Failed": "Échec de la connexion, veuillez réessayer",
+  "&AuthForm_Registration_Activation_1": "Nous vous avons envoyé un e-mail d'activation,\nqui devrait arriver bientôt.",
+  "&AuthForm_Registration_Activation_2": "Une fois l'activation de votre compte terminée,\ncliquez sur le bouton de connexion.",
+  "#AuthForm_Registration_Activation_3": "Assurez-vous de vérifier votre dossier de spam si vous ne le voyez pas.",
+  "#AuthForm_Registration_ResendActivation": "Renvoyer l'e-mail d'activation",
+  "#AuthForm_Registration_ActivationSent": "E-mail d'activation envoyé",
+  "#AuthForm_Registration_ActivationError": "Échec de la réexpédition de l'e-mail d'activation, veuillez réessayer ultérieurement",
+  "#AuthForm_FieldLabel_Email": "E-mail",
+  "#AuthForm_FieldLabel_Password": "Mot de passe",
+  "#AuthForm_FieldLabel_Username": "Nom d'utilisateur",
+  "#AuthForm_FieldLabel_TOTP": "Authentification à deux facteurs",
+  "#AuthForm_FieldDescription_TOTP": "Saisissez le code de votre application d'authentification",
+  "#AuthForm_SwitchTo_Registration": "Vous n'avez pas de compte ? S'inscrire",
+  "#AuthForm_SwitchTo_LogIn": "Vous avez déjà un compte ? Se connecter",
+  "#AuthForm_LogIn_Submit": "Se connecter",
+  "#AuthForm_Registration_Submit": "S'inscrire",
+  "#AuthForm_PasswordReset_Submit": "Mot de passe oublié ?",
+  "#AuthForm_PasswordReset_Sent": "Nous vous avons envoyé un e-mail avec les instructions de réinitialisation du mot de passe",
+  "#AuthForm_PasswordReset_Error": "Échec de la demande de réinitialisation du mot de passe, veuillez réessayer ultérieurement",
+  "#AuthForm_Back": "Retour",
+  "#AuthUI_Header": "Compte",
+  "#AuthUI_CancelAndForget": "Annuler et ne pas demander à nouveau",
+  "#AuthUI_CancelAndForget_Title": "Vous pourrez toujours le lier dans les paramètres",
+  "#ExitToDesktop": "Quitter vers le bureau",
+  "#ServerList_Name": "Nom",
+  "#ServerList_Players": "Joueurs",
+  "#ServerList_Ping": "Ping",
+  "#ServerList_Filter": "Filtrer",
+  "#ServerList_Filter_Reset": "Réinitialiser",
+  "#ServerList_Browse": "Parcourir tous les serveurs",
+  "#ServerList_Favorites": "Favoris",
+  "#ServerList_History": "Historique",
+  "#ServerList_Premium": "Supporters",
+  "#ServerList_DirectConnect": "Connexion directe",
+  "#ServerList_Tags": "Tags",
+  "#ServerList_StreamerModeWarning": "Le mode Streamer est activé - l'entrée est masquée",
+  "#ServerList_SearchHint": "Rechercher ou entrer une adresse IP",
+  "#ServerList_SearchHintWeb": "Recherche",
+  "#ServerList_SearchHint2": "Rechercher ou taper > et entrer une adresse IP",
```

Comment:
Why `>`

Reply:
It's in the english version too

Reply:
![image](https://github.com/citizenfx/fivem/assets/80094438/0bedaf11-8202-430a-a197-3d9805f5fbae)